src/routes/timelines.rs

```diff
@@ -0,0 +1,51 @@
+#![allow(dead_code)]
```

Comment:
It is because of the unimplemented routes, to avoid a few compilation warnings.

Reply:
There is not warning about them, it's actually...          dead code *badoum tss*

(no warning because functions are public, so the are maybe just used in a crate that depends on Plume, aka none)